remove hardcoding of -msse4.2 & co since it overrules -xHost when using Intel compilers
author: Kenneth Hoste (HPC-UGent)
--- tensorflow-1.15.0/tensorflow/core/BUILD.orig	2019-10-14 23:08:43.000000000 +0200
+++ tensorflow-1.15.0/tensorflow/core/BUILD	2019-11-15 09:21:33.832273690 +0100
@@ -2503,7 +2503,7 @@
     name = "lib_hash_crc32c_accelerate_internal",
     srcs = ["lib/hash/crc32c_accelerate.cc"],
     # -msse4.2 enables the use of crc32c compiler builtins.
-    copts = tf_copts() + if_linux_x86_64(["-msse4.2"]),
+    copts = tf_copts() + if_linux_x86_64([""]),
 )
 
 cc_library(
--- tensorflow-1.15.0/tensorflow/tensorflow.bzl.orig	2019-11-15 09:21:33.832273690 +0100
+++ tensorflow-1.15.0/tensorflow/tensorflow.bzl	2019-11-15 09:24:21.210549112 +0100
@@ -298,9 +298,9 @@
         if_mkl_v1_open_source_only(["-DENABLE_MKLDNN_V1"]) +
         if_enable_mkl(["-DENABLE_MKL"]) +
         if_ngraph(["-DINTEL_NGRAPH=1"]) +
-        if_android_arm(["-mfpu=neon"]) +
-        if_linux_x86_64(["-msse3"]) +
-        if_ios_x86_64(["-msse4.1"]) +
+        if_android_arm([""]) +
+        if_linux_x86_64([""]) +
+        if_ios_x86_64([""]) +
         select({
             clean_dep("//tensorflow:framework_shared_object"): [],
             "//conditions:default": ["-DTENSORFLOW_MONOLITHIC_BUILD"],
